webhook_handlers/tests/test_github.py

```diff
@@ -275,8 +275,12 @@ def test_push_updates_only_unmerged_commits_with_branch_name(self):
 
     @patch("redis.Redis.sismember", lambda x, y, z: False)
     def test_push_updates_commit_on_default_branch(self):
-        commit1 = CommitFactory(merged=False, repository=self.repo)
-        commit2 = CommitFactory(merged=False, repository=self.repo)
+        commit1 = CommitFactory(
```

Comment:
Had to change branch name for these commits to non-default branch names for the test to make sense - I believe it originally wasn't working as intended